### PR TITLE
En-5684: dummy comments for mocks and stubs

### PR DESCRIPTION
**Pull request by bogdan-rosianu**

### CODE REVIEW

File: node/mock/keyMock.go [hunk 1]
```diff
@@ -5,36 +5,40 @@ import (
...
 
+// KeyGenMock -
 type KeyGenMock struct {
 	GeneratePairMock            func() (crypto.PrivateKey, crypto.PublicKey)
 	PrivateKeyFromByteArrayMock func(b []byte) (crypto.PrivateKey, error)
 	PublicKeyFromByteArrayMock  func(b []byte) (crypto.PublicKey, error)
 	SuiteMock                   func() crypto.Suite
 }
 
-//------- PublicKeyMock
```
Comment (iulianpascalau) [inline node/mock/keyMock.go]: might preserve these placeholder

File: node/mock/keyMock.go [hunk 2]
```diff
@@ -47,20 +51,22 @@ func (sspk *PublicKeyMock) IsInterfaceNil() bool {
 	return false
 }
 
-//------- PrivateKeyMock
```
Comment (iulianpascalau) [inline node/mock/keyMock.go]: might preserve these placeholder

File: node/mock/keyMock.go [hunk 3]
```diff
@@ -73,20 +79,22 @@ func (sk *PrivateKeyStub) IsInterfaceNil() bool {
 	return false
 }
 
-//------KeyGenMock
```
Comment (iulianpascalau) [inline node/mock/keyMock.go]: might preserve these placeholder

File: process/mock/singleSignMock.go [hunk 4]
```diff
@@ -4,30 +4,34 @@ import (
...
 	SuiteCalled                  func() crypto.Suite
 }
 
+// SingleSignPublicKey -
 type SingleSignPublicKey struct {
 	SuiteCalled func() crypto.Suite
 	PointCalled func() crypto.Point
 }
 
-//------- SingleSignKeyGenMock
```
Comment (iulianpascalau) [inline process/mock/singleSignMock.go]: might preserve these placeholder

File: process/mock/singleSignMock.go [hunk 5]
```diff
@@ -40,16 +44,17 @@ func (sskgm *SingleSignKeyGenMock) IsInterfaceNil() bool {
 	return false
 }
 
-//------- SingleSignPublicKey
```
Comment (iulianpascalau) [inline process/mock/singleSignMock.go]: might preserve these placeholder

File: logger/mock/marshalizersMock.go [hunk 6]
```diff
@@ -8,15 +8,16 @@ import (
 	"github.com/golang/protobuf/proto"
 )
 
+// TestingMarshalizers -
 var TestingMarshalizers = map[string]marshal.Marshalizer{
 	"json":  &JsonMarshalizer{},
 	"proto": &ProtobufMarshalizer{},
 }
 
-//-------- Json
```
Comment (raduchis) [inline logger/mock/marshalizersMock.go]: maybe this should also remain

File: logger/mock/marshalizersMock.go [hunk 7]
```diff
@@ -39,14 +41,15 @@ func (j JsonMarshalizer) Unmarshal(obj interface{}, buff []byte) error {
 	return json.Unmarshal(buff, obj)
 }
 
+// IsInterfaceNil -
 func (j *JsonMarshalizer) IsInterfaceNil() bool {
 	return j == nil
 }
 
-//------- protobuf
```
Comment (raduchis) [inline logger/mock/marshalizersMock.go]: maybe remain

File: integrationTests/mock/keyMock.go [hunk 8]
```diff
@@ -4,25 +4,29 @@ import (
...
 
+// PrivateKeyMock -
 type PrivateKeyMock struct {
 }
 
+// KeyGenMock -
 type KeyGenMock struct {
 }
 
-//------- PublicKeyMock
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply integrationTests/mock/keyMock.go]: done

File: integrationTests/mock/keyMock.go [hunk 9]
```diff
@@ -35,20 +39,22 @@ func (sspk *PublicKeyMock) IsInterfaceNil() bool {
 	return false
 }
 
-//------- PrivateKeyMock
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply integrationTests/mock/keyMock.go]: done

File: integrationTests/mock/keyMock.go [hunk 10]
```diff
@@ -61,20 +67,22 @@ func (sk *PrivateKeyMock) IsInterfaceNil() bool {
 	return false
 }
 
-//------KeyGenMock
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply integrationTests/mock/keyMock.go]: done

File: node/mock/keyMock.go (hunk 1 above)
Comment (bogdan-rosianu) [reply node/mock/keyMock.go]: done

File: node/mock/keyMock.go (hunk 2 above)
Comment (bogdan-rosianu) [reply node/mock/keyMock.go]: done

File: node/mock/keyMock.go (hunk 3 above)
Comment (bogdan-rosianu) [reply node/mock/keyMock.go]: done

File: process/mock/singleSignMock.go (hunk 4 above)
Comment (bogdan-rosianu) [reply process/mock/singleSignMock.go]: done

File: process/mock/singleSignMock.go (hunk 5 above)
Comment (bogdan-rosianu) [reply process/mock/singleSignMock.go]: done

File: logger/mock/marshalizersMock.go (hunk 6 above)
Comment (bogdan-rosianu) [reply logger/mock/marshalizersMock.go]: done

File: logger/mock/marshalizersMock.go (hunk 7 above)
Comment (bogdan-rosianu) [reply logger/mock/marshalizersMock.go]: done

Comment (raduchis) [reply logger/mock/marshalizersMock.go]: ok

File: logger/mock/marshalizersMock.go (hunk 6 above)
Comment (raduchis) [reply logger/mock/marshalizersMock.go]: ok

**Pull request merged**